compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -55,7 +55,6 @@ class DataFormatAnnotation final : public loco::NodeAnnotation
 
 void set_data_format(loco::Node *node, const DataFormat &format)
 {
-  assert(node->annot<DataFormatAnnotation>() == nullptr);
```

Comment:
This change doesn't seem to be related with the description.

Reply:
I've made a separate PR for this. Thanks.